core/execution/future/market.go

```diff
@@ -1502,6 +1502,10 @@ func (m *Market) EnterLongBlockAuction(ctx context.Context, duration int64) {
 	if !m.canTrade() {
 		return
 	}
+	// markets that are suspended via governance, or are in monitoring auction should be unaffected by long block auctions.
```

Comment:
you want to drop the second part of the comment

Reply:
true